src/SharedLibrary/GeneratedCode/SharedLibrary_Common.h

```diff
@@ -50,6 +50,36 @@ FEATURIZER_LIBRARY_API bool DestroyErrorInfoString(/*in*/ char const *input_ptr,
 FEATURIZER_LIBRARY_API bool DestroyErrorInfo(/*in*/ ErrorInfoHandle *pHandle);
 FEATURIZER_LIBRARY_API bool DestroyTransformerSaveData(/*in*/ unsigned char const *pBuffer, /*in*/ std::size_t cBufferSize, /*out*/ ErrorInfoHandle **ppErrorInfo);
 
+FEATURIZER_LIBRARY_API_PACK_PREFIX;
+
+struct DateTimeParameter {

```

Comment:
The documentation we looked at was a plain union. This is a union within struct. I think it should still work, but I wanted to mention it just in case.

Reply:
The binary layout should still be the same, so I think that we will be all right. This is something that we should keep in mind if we start to see test failures in ML.Net.